TestResults.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Should not be pushed in the repository.
Add the file in .gitignore and moreve it from repository